db/models/userModel.js

```diff
@@ -0,0 +1,28 @@
+import { Sequelize } from 'sequelize';
+import Post from './postModel';
+
+
+let User = Sequelize.define('user', {
+  name: {
+  	Sequelize.STRING,
+  	allowNull: false
+  },
+  email: {
+  	type:Sequelize.STRING,
```

Comment:
Is it okay for line 7 to be:  'Sequelize.STRING'

Rather than:  'type: Sequelize.STRING'  ?